addons/analytic/data/analytic_data.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+
+        <record id="analytic_plan_default" model="account.analytic.plan">
+            <field name="name">Default</field>
+            <field name="company_id"></field>
+        </record>
```

Comment:
All accounts need to have a plan for the widget, as it is the plan that define which accounts are available in the widget.
And analytic accounts are automatically created for a project (if they didn't check the setting, they can't even see the analytic account), so they need a plan, and so we need this plan.
What I can do to not bug if they delete this plan is to get the first plan by id instead of this xmlid, if you think it's better